### PR TITLE
(DOCSP-23020): sync states reference page

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
New page describing the different states of the c2c utility.

Staged: https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-23020/reference/mongosync-states

### CODE REVIEW

File: source/reference/mongosync-states.txt [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.. _c2c-states:
+
+==============================
+Cluster-to-Cluster Sync States
```
Comment (alan-zheng-mdb) [inline source/reference/mongosync-states.txt:4]: can we call this page "mongosync States"

File: source/includes/api/tables/progress-response.rst [hunk 2]
```diff
@@ -9,7 +9,8 @@
 
    * - ``state``
      - string
-     - The current state of {+c2c-product-name+}.
+     - The current state of {+c2c-product-name+}. For information on the
```
Comment (alan-zheng-mdb) [inline source/includes/api/tables/progress-response.rst:12]: I'm not sure where {c2c-product-name} is, but can we use "mongosync" throughout this page?

**Review by alan-zheng-mdb — APPROVED** [review]
LGTM, the only comment is to use "mongosync" across this page instead of "Cluster-to-Cluster Sync"

File: source/reference/mongosync-states.txt [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+``mongosync`` enters different states depending on the requests it
```
Comment (davemungo) [inline source/reference/mongosync-states.txt:16]: Consider: 
``mongosync`` enters different states depending on the requests it
receives. ``mongosync`` can only be in a single state at a particular time. 
The current ``mongosync`` state determines which API operations you can use.

This page describes the ``mongosync`` states.

File: source/reference/mongosync-states.txt [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+receives. This page describes the states that ``mongosync`` can be in.
+``mongosync`` is in a single state at a particular time. The possible
+API operations you can run depend on the state of ``mongosync``.
+
+View the Current State
+----------------------
+
+To view the current state of ``mongosync``, use the
+:ref:`/progress <c2c-api-progress>`. endpoint. The endpoint returns the
```
Comment (davemungo) [inline source/reference/mongosync-states.txt:25]: Consider:

The ``/progress`` endpoint returns ...

File: source/reference/mongosync-states.txt [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+state in the ``state`` field.
+
+.. _c2c-states-descriptions:
+
+State Descriptions
+------------------
+
+The following table describes each state and the operations you can run
```
Comment (davemungo) [inline source/reference/mongosync-states.txt:33]: Consider:

The following table describes each state and lists the permitted operations.

File: source/reference/mongosync-states.txt [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+   * - ``IDLE``
+     - ``mongosync`` is initialized and ready for a sync job to
+       begin.
+     - - ``POST`` :ref:`/start <c2c-api-start>`
+       - ``GET`` :ref:`/progress <c2c-api-progress>`
+
+   * - ``RUNNING``
+     - The sync process is currently running. In this state, data is
+       initially synced to the destination cluster, and subsequent
+       writes are applied.
```
Comment (davemungo) [inline source/reference/mongosync-states.txt:53]: Consider:

cluster. Subsequent writes to the source cluster are applied to the destination cluster.

File: source/reference/mongosync-states.txt [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+   * - ``PAUSED``
+     - The sync process is paused. To resume the sync process, send a
+       request to the :ref:`/resume <c2c-api-resume>` endpoint.
+     - - ``POST`` :ref:`/resume <c2c-api-resume>`
+       - ``GET`` :ref:`/progress <c2c-api-progress>`
+
+   * - ``COMMITTING``
+     - The cutover for the sync process has started. The time it takes
+       to transition to the ``COMMITTED`` phase depends on
+       ``lagTimeSeconds``. To monitor ``lagTimeSeconds`` or see if
```
Comment (davemungo) [inline source/reference/mongosync-states.txt:67]: Consider:
To monitor ``lagTimeSeconds``, or to see

**Review by davemungo — COMMENTED** [review]
Thanks for thes updates @jeff-allen-mongo  I've left a couple of suggestions

**Pull request merged**